Modules/Product/Http/Services/ProductService.php

```diff
@@ -235,57 +217,57 @@ public function arrangeProduct($request): array
             ->orderBy('order_by', 'ASC')
             ->get();
 
-        foreach ($products as $v) {
-            $productVariations = Product::find($v->id)->productVariation()->where('status', '1')->get();
+        foreach ($products as $product) {
+            $productVariations = Product::find($product->id)->productVariations()->where('status', '1')->get();
```

Comment:
Update query here as well. User eager loading in this one too. Get the relational data using with() when you get the products.